src/System.IO.FileSystem/tests/File/Create.cs

```diff
@@ -149,19 +149,6 @@ public void OverwriteReadOnly()
             Assert.True(File.Exists(testFile));
         }
 
-        [Fact]
-        public void LongPath()
-        {
-            DirectoryInfo testDir = Directory.CreateDirectory(GetTestFilePath());
-            Assert.Throws<PathTooLongException>(() => Create(Path.Combine(testDir.FullName, new string('a', 300))));
-
-            //TODO #645: File creation does not yet have long path support on Unix or Windows
```

Comment:
This is a weird commented out blob. It should just be deleted- it should fail with segments that are this long.